src/Workspaces/CoreTestUtilities/Remote/InProcRemoteHostClientProvider.cs

```diff
@@ -29,15 +30,37 @@ public IWorkspaceService CreateService(HostWorkspaceServices workspaceServices)
                 => new InProcRemoteHostClientProvider(workspaceServices);
         }
 
+        private sealed class WorkspaceManager : RemoteWorkspaceManager
```

Comment:
InProcWorkspaceManager?  (i alwyas find it odd when subclasses have shorter names, as it makes it feel like that's teh superclass).

Reply:
This is nested type scoped only to InProcRemoteHostClientProvider.

Reply:
i understand.  but naming it with these short names still does feel unfortunate.  but if you prefer that, then that's ok.

Reply:
For me this is in the same category like naming local variables, methods or fields. Would you name a method "DoSomethingInProc" if the containing class is "InProc" already?